accounting/localizations/argentina.rst

```diff
@@ -261,11 +359,152 @@ Responsibility types:
 .. image:: media/argentina15.png
    :align: center
 
+
+Special Use Cases
+~~~~~~~~~~~~~~~~~
+
+Invoices for Services
+^^^^^^^^^^^^^^^^^^^^^
+or electronic invoices that include Services, the AFIP requires to report the  service starting
+and ending date, this information can be filled in the tab “Other Info”: 
+
+.. image:: media/argentina_edi_06.png
+   :align: center
+
+If  the dates are not selected manually before the invoice is validated, the values will be
+filled automatically considering the beginning and day of the invoice month:
+
+
+.. image:: media/argentina_edi_07.png
+   :align: center
+
+Exportation Invoices
+^^^^^^^^^^^^^^^^^^^^
+The invoices related to Exportation transactions required a Journal that used the AFIP POS 
+System “Expo Voucher - Web Service” so the proper document type be associated:
+
+.. image:: media/argentina_edi_08.png
+   :align: center
+
+When the customer selected in the Invoice has set the AFIP responsibility type as “Cliente / 
+Proveedor del Exterior” or “IVA Liberado – Ley Nº 19.640”, Odoo automatically assigned:
+
+- Journal related to the exportation Web Service.
+- Exportation document type .
+- Fiscal position: Compras/Ventas al exterior.
+- Concepto AFIP:  Products / Definitive export of goods.
+- Exempt Taxes. 
+
+.. image:: media/argentina_edi_09.png
+   :align: center
+
+.. note::
+   The Exportation Documents required the Incoterm in  “Other Info” -> Accounting section:
+   
+.. image:: media/argentina_edi_13.png
+   :align: center
+   
+   
+Fiscal Bond
+^^^^^^^^^^^
+The Electronic Fiscal bond is used for those who invoice capital goods and wish to access 
+the benefit of the Electronic Tax Bonds granted by the Ministry of Economy.
+
+For these transactions it’s important to have into consideration the next requirements:
+
+- Currency (according to parameter table) and invoice quotation.
+- Taxes.
+- Zone.
+- Detail each item.
+
+  - Code according to the Common Nomenclator of Mercosur (NCM).
+  - Complete description.
+  - Unit Net Price.
+  - Quantity.
+  - Unit of measurement.
+  - Bonus.
+  - VAT rate. 
+
+
+Electronic Credit Invoice MiPyme (FCE) 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``Invoices`` 
```

Comment:
\`\`This syntax\`\` is generally used to highlight technical terms. For important terms, we use \*\*bold\*\* as you did [here](https://www.odoo.com/documentation/user/13.0/accounting/localizations/argentina.html#introduction).

Don't forget about the non-effective line break too !